geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/GetSftCommand.scala

```diff
@@ -35,7 +42,15 @@ class GetSftCommand(parent: JCommander) extends CommandWithCatalog(parent) with
 
 object GetSftCommand {
   @Parameters(commandDescription = "Get the SimpleFeatureType of a feature")
-  class GetSftParameters extends GeoMesaConnectionParams
-    with FeatureTypeNameParam {}
+  class GetSftParameters extends GeoMesaConnectionParams with FeatureTypeNameParam {
+    @Parameter(names = Array("--concise"), description = "Render in concise format (true/false)", required = false, arity =1)
```

Comment:
how about making this arity 0 so you specify '--concise' instead of '--concise true' ? (i think that's the way arity works...)